k2/csrc/fsa_equivalent.h

```diff
@@ -32,6 +32,10 @@ bool IsRandEquivalent(const Fsa &a, const Fsa &b, std::size_t npath = 100);
   @param [in]  a_weights  Arc weights of `a`
   @param [in]  b          The other FSA to be checked the equivalence
   @param [in]  b_weights  Arc weights of `b`
+  @param [in]  beam       beam > 0 that affects pruning in `a`; the algorithm
+                          will keep paths that are within `beam` of the best
+                          path in `a` to check the equivalence. Just keep
+                          `kFloatInfinity` if you don't want pruning.
   @param [in]  top_sorted If both `a` and `b` are topological sorted or not.
```

Comment:
Comment here is not very clear.  I'd say:

`The user may set this to true if both `a` and `b` are topologically sorted; this makes this
  function faster.  Otherwise it must be set to false.`

Reply:
OK, noted that now we don't implement the version for non-sorted fsa, so I will keep `CHECK(top_sorted)` in the cc file.